versioned_docs/version-0.10/enterprise/administration/configuration.mdx

```diff
@@ -195,6 +195,8 @@ cloud:
       enabled: true
       bucket: "Bucket Name"
       project: "Project ID"
+      workloadIdentity:
```

Comment:
@maroshii please move this to https://gith
ub.com/okteto/docs/blob/main/src/content/enterprise/install/releases.mdx, since this doesn't work on OE 0.10.